### PR TITLE
[RW-3104][RW-3106] Add Google Analytics to Cohort Review

### PR DESCRIPTION
**Pull request by dolbeew**
Events based on the Google doc tabs for [Review List](https://docs.google.com/spreadsheets/d/1V_Zp0ZtN8f237LNcaGQA-WZzYJu9g8ECq80gJCV8njA/edit?ts=5d4052a7#gid=733768841) and [Review Individual](https://docs.google.com/spreadsheets/d/1V_Zp0ZtN8f237LNcaGQA-WZzYJu9g8ECq80gJCV8njA/edit?ts=5d4052a7#gid=1011708973)

### CODE REVIEW

File: ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx [hunk 1]
```diff
@@ -76,9 +77,8 @@ export const CreateReviewModal = withCurrentWorkspace()(
       };
     }
 
-    cancelReview() {
```
Comment (gjuggler) [inline ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx]: (No action required) These deletion seems interesting – is this just a drive-by cleanup?

File: ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx [hunk 2]
```diff
@@ -76,9 +77,8 @@ export const CreateReviewModal = withCurrentWorkspace()(
...
-      const {workspace: {id, namespace}} = this.props;
-      navigate(['workspaces', namespace, id, 'data', 'cohorts']);
+    componentDidMount(): void {
+      triggerEvent('Create Review Set', 'Click', 'Create Set Review');
```
Comment (gjuggler) [inline ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx]: Nitpick: is there a reason for the different phrases "Create Review Set" and "Create Set Review"?

File: ui/src/app/cohort-review/detail-header/detail-header.component.tsx [hunk 3]
```diff
@@ -399,8 +411,8 @@ export const DetailHeader = withCurrentWorkspace()(
         {errors && <div className='error-messages'>
           <ValidationError>
             {summarizeErrors(errors && (
-              (filterTab === 'age' && ((ageMin && errors.ageMin) || (ageMax && errors.ageMax))) ||
-              (filterTab === 'date' && ((dateMin && errors.dateMin) || (dateMax && errors.dateMax)))
+              (filterTab === 'Age' && ((ageMin && errors.ageMin) || (ageMax && errors.ageMax))) ||
```
Comment (gjuggler) [inline ui/src/app/cohort-review/detail-header/detail-header.component.tsx]: The repetition of bare strings that have functional importance is a bit of a code smell. I'd lean towards defining 'Date' and 'Age' as constant strings at the top level, e.g. const AGE_FILTER_KEY = 'Age';

File: ui/src/app/cohort-review/table-page/table-page.tsx [hunk 4]
```diff
@@ -411,6 +412,7 @@ export const ParticipantsTable = withCurrentWorkspace()(
     }
 
     goBack() {
+      triggerEvent('Review Participant List', 'Click', 'Back to cohort - Review Participant List');
```
Comment (gjuggler) [inline ui/src/app/cohort-review/table-page/table-page.tsx]: Optional: the "Review Participant List' string is used in a few places, you could factor that out into a const PAGE_NAME

File: ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx (hunk 1 above)
Comment (dolbeew) [reply ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx]: Yep, this was an unused function that should have been removed previously.

File: ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx (hunk 2 above)
Comment (dolbeew) [reply ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx]: I actually mistyped this. The label should be "Create Set - Review,"  which I believe uses the format seen in some other events of "{action} - {location}"

File: ui/src/app/cohort-review/detail-header/detail-header.component.tsx (hunk 3 above)
Comment (dolbeew) [reply ui/src/app/cohort-review/detail-header/detail-header.component.tsx]: Good idea, done.

File: ui/src/app/cohort-review/table-page/table-page.tsx (hunk 4 above)
Comment (dolbeew) [reply ui/src/app/cohort-review/table-page/table-page.tsx]: Factored out this string as well as several in `detail-header` and `detail-tabs` components.
